Source/WebInspectorUI/UserInterface/Views/NavigationSidebarPanel.js

```diff
@@ -633,20 +635,8 @@ WI.NavigationSidebarPanel = class NavigationSidebarPanel extends WI.SidebarPanel
 
     _checkForStaleResourcesIfNeeded()
     {
-        if (!this._checkForStaleResourcesTimeoutIdentifier || !this._shouldAutoPruneStaleTopLevelResourceTreeElements)
-            return;
-        this.pruneStaleResourceTreeElements();
-    }
-
-    _checkForStaleResources(event)
-    {
-        console.assert(this._shouldAutoPruneStaleTopLevelResourceTreeElements);
-
-        if (this._checkForStaleResourcesTimeoutIdentifier)
-            return;
-
-        // Check on a delay to coalesce multiple calls to _checkForStaleResources.
-        this._checkForStaleResourcesTimeoutIdentifier = setTimeout(this.pruneStaleResourceTreeElements.bind(this));
+        if (this._shouldAutoPruneStaleTopLevelResourceTreeElements && !this._checkForStaleResourcesTimeoutIdentifier)
```

Comment:
NIT: i think `setTimeout` can return `0` so it's safest to check for `=== undefined`